truss/environment_inference/requirements_inference.py

```diff
@@ -49,9 +52,17 @@ def infer_deps(must_include_deps: Set[str] = None) -> Set[str]:
     )
 
     # Cross-check the names of installed packages vs. imported packages to get versions
-    for m in pkg_resources.working_set:
-        if m.project_name in imports and m.project_name not in IGNORED_PACKAGES:
-            requirements.add(f"{m.project_name}=={m.version}")
+    for pkg_in_frame in pkg_resources.working_set:
+        if (
+            pkg_in_frame.project_name in imports
+            and pkg_in_frame.project_name not in IGNORED_PACKAGES
+        ):
+            requirements.add(f"{pkg_in_frame.project_name}=={pkg_in_frame.version}")
+            if pkg_in_frame.project_name in must_include_dependencies:
```

Comment:
Good to leave a comment here explaining the removal. Why doesn't set union take care of removing the duplicates? must_include_deps and must_include_dependies are so similar, I can't tell what is each for. Would be good to simplify this code.